README.md

```diff
@@ -6,6 +6,78 @@
 ![Screenshots with my stupid grinning face](images/screenshot.jpg)
 (Credits for the nice backgrounds to [Mary Sabell](https://dribbble.com/shots/4686178-Bauhaus-Poster) and [PhotoFunia](https://photofunia.com/effects/retro-wave))
 
+## Maintainers
+
+  * Phil Ashby (@phlash)
+  * Benny Baumann (@benbe)
+  * Florian Echtler (@floe)
+
+## Building
+
+Install dependencies (`sudo apt install libopencv-dev build-essential v4l2loopback-dkms curl`).
+
+Clone this repository with `git clone --recursive https://github.com/floe/backscrub.git`.
+To speed up the checkout you can additionally pass `--depth=1` to `git clone`.
+This is okay, if you only want to download and build the code, however, for development it is not recommended.
+
+Use `cmake` to build the project: create a subfolder (e.g. `build`), change to that folder and run: `cmake && make -j4` (please replace "4" with the number of your cores).
```

Comment:
I usually work in the source folder with:
```bash
% cmake -B build
% cmake --build build
```
but that's personal pref/taste. If using the `cd` approach, then the parent directory needs to be referenced ie:
`cmake .. && make -j <n>`

Reply:
ACK on the `cd` approach.